src/main/java/teammates/common/util/Const.java

```diff
@@ -977,6 +982,7 @@ public static final String getjQueryUiFilePath(boolean isDevServer) {
         public static final String ADMIN_EMAIL_CREATE_GROUP_RECEIVER_LIST_UPLOAD_URL =
                 "/admin/adminEmailCreateGroupReceiverListUploadUrl";
         
+        public static final String PUBLIC_FILE_SERVE = "/public/publicImageServe";
```

Comment:
`PUBLIC_IMAGE_SERVE` is a better name, even though the constant below has `FILE` in place of `IMAGE`.